ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-bgp.j2

```diff
@@ -943,6 +943,67 @@ router bgp {{ router_bgp.as }}
+{%         endif %}
+{%         for peer_group in router_bgp.address_family_path_selection.peer_groups | arista.avd.natural_sort('name') %}
+{%             if peer_group.activate is arista.avd.defined(true) %}
+      neighbor {{ peer_group.name }} activate
+{%             elif peer_group.activate is arista.avd.defined(false) %}
+      no neighbor {{ peer_group.name }} activate
+{%             endif %}
+{%             if peer_group.additional_paths.receive is arista.avd.defined(true) %}
+      neighbor {{ peer_group.name }} additional-paths receive
+{%             endif %}
+{%             if peer_group.additional_paths.send.any is arista.avd.defined(true) %}
+      neighbor {{ peer_group.name }} additional-paths send any
+{%             elif peer_group.additional_paths.send.backup is arista.avd.defined(true) %}
+      neighbor {{ peer_group.name }} additional-paths send backup
+{%             elif peer_group.additional_paths.send.ecmp is arista.avd.defined(true) %}
+      neighbor {{ peer_group.name }} additional-paths send ecmp
+{%             elif peer_group.additional_paths.send.ecmp_limit is arista.avd.defined %}
+      neighbor {{ peer_group.name }} additional-paths send ecmp limit {{ peer_group.additional_paths.send.ecmp_limit }}
+{%             elif peer_group.additional_paths.send.limit is arista.avd.defined %}
+      neighbor {{ peer_group.name }} additional-paths send limit {{ peer_group.additional_paths.send.limit }}
+{%             endif %}
+{%         endfor %}
+{%         for neighbor in router_bgp.address_family_path_selection.neighbors | arista.avd.natural_sort('ip_address') %}
+{%             if neighbor.activate is arista.avd.defined(true) %}
+      neighbor {{ neighbor.ip_address }} activate
+{%             elif neighbor.activate is arista.avd.defined(false) %}
+      no neighbor {{ neighbor.ip_address }} activate
+{%             endif %}
+{%             if neighbor.additional_paths.receive is arista.avd.defined(true) %}
+      neighbor {{ neighbor.ip_address }} additional-paths receive
+{%             endif %}
+{%             if neighbor.additional_paths.send.any is arista.avd.defined(true) %}
+      neighbor {{ neighbor.ip_address }} additional-paths send any
+{%             elif neighbor.additional_paths.send.backup is arista.avd.defined(true) %}
+      neighbor {{ neighbor.ip_address }} additional-paths send backup
+{%             elif neighbor.additional_paths.send.ecmp is arista.avd.defined(true) %}
+      neighbor {{ neighbor.ip_address }} additional-paths send ecmp
+{%             elif neighbor.additional_paths.send.ecmp_limit is arista.avd.defined %}
+      neighbor {{ neighbor.ip_address }} additional-paths send ecmp limit {{ neighbor.additional_paths.send.ecmp_limit }}
+{%             elif neighbor.additional_paths.send.limit is arista.avd.defined %}
+      neighbor {{ neighbor.ip_address }} additional-paths send limit {{ neighbor.additional_paths.send.limit }}
+{%             endif %}
```

Comment:
This doesn't look right. I would assume that at least some of these commands can be there side my side or maybe even as add-ons on a single line.

The limits are probably separate command lines.

If the others are mutually exclusive we should change the model to have `send: <any | backup | ecmp>` since only one of them can be true.

```yaml
...
send: < any | backup | ecmp >
send_ecmp_limit: int
send_limit: int
```

Reply:
as discussed - ignoring this becasue we already have this exact same model in our schema and it does not make sense to have multiple ways to represent the same knobs.